nixos/tests/mysql/mysql.nix

```
@@ -172,32 +172,32 @@ import ./../make-test-python.nix ({ pkgs, ...} : {
         "echo 'use testdb; select test_id from tests;' | sudo -u testuser mysql -u testuser -N | grep 42"
     )
 
-    # Check if TokuDB plugin works
+    # Check if RocksDB plugin works
     mariadb.succeed(
-        "echo 'use testdb; create table tokudb (test_id INT, PRIMARY KEY (test_id)) ENGINE = TokuDB;' | sudo -u testuser mysql -u testuser"
+        "echo 'use testdb; create table rocksdb (test_id INT, PRIMARY KEY (test_id)) ENGINE = RocksDB;' | sudo -u testuser mysql -u testuser"
     )
     mariadb.succeed(
-        "echo 'use testdb; insert into tokudb values (25);' | sudo -u testuser mysql -u testuser"
+        "echo 'use testdb; insert into rocksdb values (28);' | sudo -u testuser mysql -u testuser"
     )
     mariadb.succeed(
-        "echo 'use testdb; select test_id from tokudb;' | sudo -u testuser mysql -u testuser -N | grep 25"
+        "echo 'use testdb; select test_id from rocksdb;' | sudo -u testuser mysql -u testuser -N | grep 28"
     )
     mariadb.succeed(
-        "echo 'use testdb; drop table tokudb;' | sudo -u testuser mysql -u testuser"
+        "echo 'use testdb; drop table rocksdb;' | sudo -u testuser mysql -u testuser"
     )
-
-    # Check if RocksDB plugin works
+  '' + pkgs.stdenv.lib.optionalString (!pkgs.stdenv.hostPlatform.isAarch32 && !pkgs.stdenv.hostPlatform.isAarch64) ''
```

We should better check for architectures supported, not those not supported - TokuDB probably also doesn't run on riscv5 ;)

```suggestion
  '' + pkgs.stdenv.lib.optionalString pkgs.stdenv.isx86_64 ''
```

Also, shouldn't this be pkgs.stdenv.

```suggestion
  '' + pkgs.stdenv.lib.optionalString (!pkgs.stdenv.hostPlatform.isAarch32 && !pkgs.stdenv.hostPlatform.isAarch64) ''
```

---

Fixed.